test/test_client.py

```diff
@@ -1591,6 +1591,25 @@ def test_network_error_message(self):
             with self.assertRaisesRegex(AutoReconnect, expected):
                 client.pymongo_test.test.find_one({})
 
+    def test_process_periodic_tasks(self):
+        client = rs_or_single_client()
+        coll = client.db.collection
+        coll.insert_many([{} for _ in range(5)])
+        cursor = coll.find(batch_size=2)
+        cursor.next()
+        c_id = cursor.cursor_id
+        assert c_id
+        client.close()
+        # Add cursor to kill cursors queue
+        del cursor
+        wait_until(lambda: c_id in [c for _, c, _ in
+                                    client._MongoClient__kill_cursors_queue],
+                   "waited for cursor to be added to queue")
+        try:
+            client._process_periodic_tasks()  # This must not raise or print any exceptions
+        except Exception:
+            self.fail("client._process_periodic_tasks() raised an exception")
+
 
```

Comment:
We don't need the try/except/fail here. Just call client._process_periodic_tasks(). If that raises unexpectedly the teat will fail automatically.

Reply:
Done.